purchase_default_terms_conditions/models/res_config_settings.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 - TODAY, Marcel Savegnago - Escodoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+
+    _inherit = "res.config.settings"
+
+    purchase_note = fields.Text(
+        related="company_id.purchase_note",
+        string="Purchase Terms & Conditions",
+        default_model="purchase.order",
```

Comment:
`default_model="purchase.order",` this is not needed now as we are not going to create Defaults from here.

Reply:
nice catch @Swapnesh-SerpentCS ! [Here is the doc behind this](https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/models/res_config.py#L344).
since `default_note` is renamed in `purchase_note`, the `default_model` key parameter is not needed anymore